sanic/app.py

```diff
@@ -1177,6 +1178,7 @@ def run(
 
         try:
             self.is_running = True
+            self.is_stopping = False
```

Comment:
What's the diff between `is_running` and `is_stopping` ?

Reply:
When the app has not yet started, both are False. After calling `app.stop` the app may keep running for a while, so in that period both are True. Asyncio implementation uses `loop._stopping` in a similar fashion but uvloop doesn't have that and depending on private parts of the implementation is ugly in any case.